services/wireguard/service.go

```diff
@@ -27,35 +28,46 @@ import (
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/node/money"
 	dto_discovery "github.com/mysteriumnetwork/node/service_discovery/dto"
+	"github.com/mysteriumnetwork/node/services/wireguard/resources"
 	"github.com/mysteriumnetwork/node/session"
 )
 
 const logPrefix = "[service-wireguard] "
 
-// ErrAlreadyStarted is the error we return when the start is called multiple times
+// ErrAlreadyStarted is the error we return when the start is called multiple times.
 var ErrAlreadyStarted = errors.New("Service already started")
 
 // NewManager creates new instance of Wireguard service
 func NewManager(locationResolver location.Resolver, ipResolver ip.Resolver) *Manager {
 	return &Manager{
 		locationResolver: locationResolver,
 		ipResolver:       ipResolver,
+		resources:        resources.Handler{},
 	}
 }
 
-// Manager represents entrypoint for Wireguard service
+// Manager represents entrypoint for Wireguard service.
 type Manager struct {
-	process          sync.WaitGroup
-	locationResolver location.Resolver
-	ipResolver       ip.Resolver
-	isStarted        bool
+	isStarted          bool
+	process            sync.WaitGroup
+	locationResolver   location.Resolver
+	ipResolver         ip.Resolver
+	connectionEndpoint ConfigProvider
```

Comment:
connectionEndpoint but its actually ConfigProvider? So it's connection or config provider? Inconsistent naming

Reply:
No more ConfigProvider here.